GitUI/UserControls/BranchComboBox.cs

```diff
@@ -95,6 +104,13 @@ private void selectMultipleBranchesButton_Click(object sender, EventArgs e)
             }
 
             branches.Text = branchesText;
+
+            SelectedValueChanged?.Invoke(this, EventArgs.Empty);
```

Comment:
[nit] Since there are now two invocations there's a small risk these callsites go out of sync. That's why to avoid such situations a common method like the following is typically created:
```cs
private void OnSelectedValueChanged(EventArgs e) => SelectedValueChanged?.Invoke(this, e);
```

I'm not requesting a change just highlighting a possible issue.